src/posts/2020-10-28-dyslexia-fonts.md

```diff
@@ -0,0 +1,49 @@
+---
+layout: post
+title: Dyslexia fonts
+description: A quick overview of dyslexia fonts.
+category: Assistive technology
+author: Hamsa Harcourt
+date: 2020-10-28
+eleventyNavigation:
+  key: {{ category }}
+  title: {{ title }}
+tags:
+  - assistivetechnology
+---
+
+According to Oxford Advanced Learner's Dictionary:
+
+> Dyslexia is a slight disorder of the brain that causes difficulty in reading and spelling, for example, but does not affect intelligence
+
+Dyslexia affects up to 1 in 5 people. However, dyslexia like most conditions, ranges from a mild disability to an extremely disabling one. Since people with dyslexia transpose characters, fonts are an important facet to consider.
```

Comment:
I love these feedbacks. They're really helpful. 

What do you think about me changing "People with dyslexia" to "dyslexics"

Reply:
Again, I find myself using "letters, numbers, punctuation, and other characters" throughout the page. Do you think this will pose a challenge to our readers?

What do you think about informing our readers beforehand that characters refers to letters, numbers, punctuation etc

Reply:
>What do you think about me changing "People with dyslexia" to "dyslexics"

I'm unsure. I think investigating what the preferred, identity-first language is for this community, as "dyslexics" could be seen as derogatory. 

>What do you think about informing our readers beforehand that characters refers to letters, numbers, punctuation etc

I think that could work. Something like "written characters (letters, numbers, punctuation, and other symbols)" could work. What I want to stress here is that it's more than just the alphabet that is affected.